rotkehlchen/db/ethtx.py

```diff
@@ -420,3 +420,12 @@ def get_queried_range(self, address: ChecksumEthAddress) -> Tuple[Timestamp, Tim
             ends.append(tx_range[1])
 
         return max(starts), min(ends)
+
+    def get_max_trace_id(self, txn_hash: str) -> int:
+        """Get the max trace id from the database"""
+        cursor = self.db.conn.cursor()
+        trace_id, = cursor.execute(
+            'SELECT MAX(trace_id) from ethereum_internal_transactions WHERE parent_tx_hash=?',
+            (txn_hash,),
+        ).fetchone()
+        return trace_id if trace_id else 0
```

Comment:
Why? Why are you doing it? Why you don't want to return `None` ? Returning `None` is a much more clear sign that there is no `trace_id` in the database than `0`.